mpqp/execution/result.py

```diff
@@ -59,14 +58,23 @@ def amplitudes(self):
         """Return the amplitudes of the state vector"""
         return self.vector
 
+    
     def __str__(self):
-        cleaned_vector = str(self.vector).replace("\n", " ")
-        cleaned_probas = str(self.probabilities).replace("\n", " ")
-        return f""" State vector: {cleaned_vector}
- Probabilities: {cleaned_probas}
+        return f""" State vector: {_clean_array(self.vector)}
+ Probabilities: {_clean_array(self.probabilities)}
  Number of qubits: {self.nb_qubits}
 """
 
+def _clean_array(array): # type: ignore
```

Comment:
Did you check that all examples in docstrings were update with this output formatting ?